lib/src/blocs/app/notification_state.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:equatable/equatable.dart';
+
+class NotificationState extends Equatable {
+  NotificationState();
+  @override
+  List<Object?> get props => [];
+}
+
+class NotificationStatusState extends NotificationState {
```

Comment:
status state is a bit ambiguous, something like `notificationsonstate` and `notificationoffstate` makes more sense

Reply:
Can I call it NotificationToggleState since you will be toggling the notification on and off?